src/Libraries/RevitNodes/Elements/Views/View3DTemplate.cs

```diff
@@ -0,0 +1,60 @@
+﻿using DynamoServices;
+using System;
+
+namespace Revit.Elements.Views
+{
+    /// <summary>
+    /// A Revit 3D View Template
+    /// </summary>
+    [RegisterForTrace]
+    public class View3DTemplate : View3D
+    {
+
+        #region Private constructors
+
+        /// <summary>
+        /// Create a Revit Floor Plan from Autodesk View Plan
+        /// </summary>
+        private View3DTemplate(Autodesk.Revit.DB.View3D view)
+        {
+            SafeInit(() => Init3DView(view), true);
+        }
+
+        #endregion
+
+        #region Helpers for private constructors
+
+        /// <summary>
+        /// Initialize a FloorPlanView element
+        /// </summary>
+        private void Init3DView(Autodesk.Revit.DB.View3D view)
+        {
+            InternalSetView3D(view);
+        }
+
+        #endregion
+
+        #region Internal methods
+
+        /// <summary>
+        /// Create from existing Element
+        /// </summary>
+        /// <param name="plan"></param>
+        /// <param name="isRevitOwned"></param>
+        /// <returns></returns>
+        internal static View3DTemplate FromExisting(Autodesk.Revit.DB.View3D plan, bool isRevitOwned)
```

Comment:
It would have been nice to name the first parameter `view` instead of `plan` for two reasons:
- consistency with other view classes
- `plan` is kind of antithesis of 3D when it comes to views

Reply:
Absolutely agree! Addressed with https://github.com/DynamoDS/DynamoRevit/pull/2926, @Mikhinja